src/core.cpp

```diff
@@ -3,55 +3,112 @@
 #include <random>
 
 #include "core.h"
+#include "action.h"
 
-double Character::GetMoney() {
-    return this->money_;
+RelationshipModifier::RelationshipModifier(Character* target, int start_date, int end_date, double opinion_modifier_) {
+    this->target_ = target;
+    this->start_date_ = start_date;
+    this->end_date_ = end_date;
+    this->opinion_modifier_ = opinion_modifier_;
 }
 
-void Character::SetMoney(double money) {
+Character::Character(int id, double money) {
+    this->id_ = id;
     this->money_ = money;
 }
 
-
-Action::Action(Character* actor, double score) {
-    this->actor_ = actor;
-    this->score_ = score;
+int Character::GetId() const {
+    return this->id_;
```

Comment:
Simple functions like this I'd normally put in the header so they can be inlined.

Reply:
doing getters/setters, even trivial ones, is basically the "way it's done" in Java. Is that the case in c++?